scripts/rlog/pr.fish

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env fish
+set -g SCRIPTS (dirname (dirname (status -f)))
+source $SCRIPTS/lib/tests.fish
+
+################################################################################
+## load tests definition from ArangoDB
+################################################################################
+
+echo "Using test definitions from arangodb repo"
+
+ls -l
+echo $SCRIPTS
+set -l TESTS
+source $INNERWORKDIR/ArangoDB/tests/Definition/rlog/pr.fish
```

Comment:
There is a need to have `test -f $INNERWORKDIR/ArangoDB/tests/Definition/rlog/pr.fish` prior to source (not all branches may have it and `exit 1` in case of absence.